src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -93,6 +95,19 @@ public override async Task<int> ExecuteAsync()
                             return Constants.ErrorCode;
                         }
                     }
+                    if (rootBuilds.Contains(build))
+                    {
+                        try
+                        {
+                            downloadedBuild.Dependencies = await GetBuildDependenciesAsync(build);
+                        }
+                        catch (DependencyFileNotFoundException)
```

Comment:
This "catch", and the "how to run tests doc" are the only delta from before reverting.